gp-includes/things/translation-set.php

```diff
@@ -314,13 +316,13 @@ public function import( $translations, $desired_status = 'current' ) {
 				/**
 				 * Filter the the status of imported translations of a translation set.
 				 *
-				 * @since 1.0.0
+				 * @since 2.2.0
 				 *
-				 * @param string            $status The status of imported translations.
+				 * @param string $status The status of imported translations.
 				 * @param Translation_Entry $entry  Translation entry object to import.
+				 * @param GP_Translation|false $translated The previous translation.
 				 */
-				$entry->status = apply_filters( 'gp_translation_set_import_status', $entry->status, $entry );
-
+				$entry->status = apply_filters( 'gp_translation_set_import_status', $entry->status, $entry, $translated );
```

Comment:
The argument `$entry` (see below) was added in 04484247ea1a018b6635f364532dfc2f39261478 but only in one of the occurrences of the filter.